docs/source/pages/community/roadmap.md

```diff
@@ -0,0 +1,22 @@
+# Roadmap
```

Comment:
This is good, but I would add some more detail and timings. To me, a roadmap isn't much use if I don't have an idea of when the feature will be implemented. Everyone is on a deadline so often (after your funding ends) == never.

Reply:
Thanks @adamltyson as we touched on before, I decided to split datashuttle and neuroblueprint roadmap as some things are not really related (e.g. automated conversion to NWB is not relevant to NB). Please see here and a draft [at this PR](https://github.com/neuroinformatics-unit/NeuroBlueprint/pull/58).

Reply:
Yes I agree, we can possibly add a section on [more datatypes](https://github.com/neuroinformatics-unit/NeuroBlueprint/issues/57) when that is a bit more formalised.